config/routes.rb

```diff
@@ -2,266 +2,268 @@
 
 # require "rswag-api"
 
-require "sidekiq/web"
```

Comment:
Did you mean to get rid of this?

Reply:
Yeah, moving it to the engine routes instead